.github/workflows/auto_release.yml

```diff
@@ -0,0 +1,90 @@
+name: "Auto release"
+
+on:
+  workflow_dispatch:
+
+env:
+  HONEYCOMB_WRITEKEY: zEepjMgnVKilj943ApiKvH 
```

Comment:
Hmm... so I just followed the integration guide on this here. Puppet modules seems to just put the key there:

https://github.com/puppetlabs/puppetlabs-motd/blob/main/.github/workflows/pr_test.yml#L8
https://github.com/puppetlabs/puppetlabs-mysql/blob/main/.github/workflows/pr_test.yml#L8
https://github.com/puppetlabs/puppetlabs-apt/blob/main/.github/workflows/pr_test.yml#L8

So I assumed that this would be the way to go... I will have to look into how to get this variable from there into the github action while still not modifying the template: https://github.com/puppetlabs/pdk-templates/blob/main/moduleroot/.github/workflows/pr_test.yml.erb#L9

Reply:
Maybe I'm missing or misunderstanding something. But can't I just take their API key and send arbitrary data to their datasets?

Reply:
TBH, I have not looked into honeycomb enough to really understand what it possible there and exactly what it does and why it is useful. Currently I am also not really sure if I will ever look into the data it collects. I just wanted to be as close as possible to the official modules and what the PDK does in order to make maintenance as easy as possible. And if just deactivating it causes the PDK to generate broken gitlab action configs, than I have no choice but to enable it for now.

So if someone for some reason is uploading entire movies via that key to honeycomb and starts sharing it with others, be my guest I guess ;)

Reply:
I am now using the github secret instead of putting it directly into the files.

Reply:
So I also split up the changes into multiple separate commits. This will also make review easier, because it allows to see what files are generated and what I have manually changed.